.github/ISSUE_TEMPLATE/01-github-request---onboarding.yml

```diff
@@ -26,6 +26,11 @@ body:
         * ...
     validations:
       required: true
+  - type: input
+    id: squad
+    attributes:
+      label: User Squad
+      description: If the users specified above are members of a named squad or team at your firm, optionally specify it here.  This will help simplify access management in the future.
```

Comment:
Should we make the distinction between squad and Github team?  Does it matter if folks specify a Github team, here?

Reply:
I don't know - probably a question for Ned

Reply:
TBH, i don't know what information the 2U help desk will have when they fill this out.  This seems like a fine description to start with.